source/language/openpulse.rst

```diff
@@ -274,13 +282,13 @@ The following are possible parameters that might be included:
 Again it is up to the hardware vendor to determine the parameters and write a
```

Comment:
On line 274, says "only required parameter is a `frame`". Should update to include the channel.